### ISSUE TITLE
Support or Document retryTimes

### ISSUE THREAD

**Issue by Vandivier** [palmerhq/cypress-image-snapshot#118]
The documentation currently states `Additionally, any options for cy.screenshot() and jest-image-snapshot can be passed in`, but this is ambiguous with respect to [`jest. retryTimes`](https://github.com/americanexpress/jest-image-snapshot#jestretrytimes) for at least three reasons:

1. `retryTimes` is not an options for Jest's `toMatchImageSnapshot` method, and it's not super clear that it's an option at all in the sense of an option object property. It's usually invoked as a standalone method.
2. [`retryTimes` docs](https://jestjs.io/docs/en/jest-object#jestretrytimes) warn us that it only works with jest-circus, so it may be specially unsupported compared to other options.
3. [Cypress docs](https://docs.cypress.io/api/commands/screenshot.html#Assertions) specify that `cy.screenshot` will not retry, so it's not clear whether this is a special case where retry doesn't work even using cypress-image-snapshot.

As a supplementary comment, I'm using `cypress-plugin-retries` to support retrying in a normal test scenario, but it isn't respected when I attempt to update my baseline image snapshots because that isn't considered a fail. I don't have the expectation that these two plugins need to support one another necessarily, but it sure would be nice.

In summary, if retrying is supported I would request clearer documentation on how to do that. If it's not supported, I would request it as a feature. Potentially through re-using `cypress-plugin-retries` or maybe through some other way.
